frontend/lib/src/dataframes/Quiver.ts

```diff
@@ -123,68 +123,114 @@ type IntervalClosed = "left" | "right" | "both" | "neither"
 type IntervalType = `interval[${IntervalData}, ${IntervalClosed}]`
 
 // The frequency strings defined in pandas.
-// See: https://pandas.pydata.org/docs/user_guide/timeseries.html#dateoffset-objects
-type SupportedPandasOffsetType = "W" | "Q" | "D" | "H" | "T" | "S" | "L"
+// See: https://pandas.pydata.org/docs/user_guide/timeseries.html#period-aliases
+// Not supported: "N" (nanoseconds), "U" & "us" (microseconds), and "B" (business days).
+// Reason is that these types are not supported by moment.js, but also they are not
+// very commonly used in practice.
+type SupportedPandasOffsetType =
+  // yearly frequency:
+  | "A" // deprecated alias
+  | "Y"
+  // quarterly frequency:
+  | "Q"
+  // monthly frequency:
+  | "M"
+  // weekly frequency:
+  | "W"
+  // calendar day frequency:
+  | "D"
+  // hourly frequency:
+  | "H" // deprecated alias
```

Comment:
is "H" a deprecated alias or "h"?
Should it be:
```
// hourly frequency:
| "H"
// deprecated alias
"h"
```
?

Reply:
`H` is deprecated. The comments are always in the same line of the deprecated alias